src/components/Courses/components/CourseCard/CourseCard.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import Button from '../../../../common/Button'
+import { StyledCourseCard } from '../../../../common/CommonHTML'
+import { mockedAuthorsList } from '../../../../constants'
+import getDuration from '../../../../helpers/getCourseDuration'
+
+export default function CourseCard(props) {
+  const {
+    title, description, creationDate, duration, authors
+  } = props.item
+  const formatCreation = creationDate.split('/').join('.')
+  let authorArr = []
+  authors.forEach((element) => {
+    mockedAuthorsList.filter((item) => {
+      if (item.id === element) authorArr.push(item.name)
+    })
+  })
+  const authorList = authorArr.map((item, index) => {
+    let span = ''
+    if (index !== 0) {
+      span = `, ${item}`
+    } else {
+      span = `${item}`
+    }
+    return (
+      <span key={item}>{span}</span>
+    )
+  })
```

Comment:
This part is component rendering, we could
```js

  const renderAuthor = (author, key) => {
    const prefix = key === 0 ? ', ' : '';
    return <span key={author}>{`${prefix}${item}`}</span>
  }

  const renderAuthorsList = (authorsList) => {
    return authorsList.map((author, key) => renderAuthor(author, key));
  };
```

Reply:
I did not test the code, it's for communication purpose only, please debug it later. :)